aldryn_config.py

```diff
@@ -4,8 +4,8 @@
 
 class Form(forms.BaseForm):
 
-    show_all_recipients = forms.CheckboxField('Show all users in form recipients select', required=False)
+    show_all_recipients = forms.CheckboxField('Include only admin users as selectable e-mail recipients instead of everyone.', required=False)
 
     def to_settings(self, data, settings):
-        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = data['show_all_recipients']
+        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = not data['show_all_recipients']
```

Comment:
as discussed in hangout: switch the label to ``Show all users as selectable e-mail recipients (not only admins)`` and make it checked by default.